Taggers/python/flashggDifferentialPhoIdInputsCorrection_cfi.py

```diff
@@ -54,7 +54,10 @@ def setup_flashggDifferentialPhoIdInputsCorrection( process, metaConditions ):
     setattr(flashggDifferentialPhoIdInputsCorrection, "photonIdMVAweightfile_EE", cms.FileInPath(str(metaConditions['flashggPhotons']['photonIdMVAweightfile_EE'])))
     setattr(flashggDifferentialPhoIdInputsCorrection, "effAreasConfigFile", cms.FileInPath(str(metaConditions['flashggPhotons']['effAreasConfigFile'])))
     setattr(flashggDifferentialPhoIdInputsCorrection, "is2017", cms.bool(metaConditions['flashggPhotons']['is2017']))
-
+    flashggTHQLeptonicTag.thqCatweightfile_ForNonPeakingBkg = cms.FileInPath(str(metaConditions['THQLeptonicTag']['MVAweights_VsAllBkg']))
```

Comment:
This should not be implemented here but inside flashggTagSequence_cff

Reply:
It is done. Now I implemented inside flashggTagSequence_cfi

Reply:
For indentation I rerun https://github.com/cms-analysis/flashgg/blob/dev_legacy_runII/setup_flashgg.sh#L65 manually. But indentation are still not proper.